BUILD

```diff
@@ -539,13 +539,103 @@ internal_gen_well_known_protos_java(
     srcs = WELL_KNOWN_PROTOS,
 )
 
+JAVA_LITE_FILES = [
```

Comment:
Why not inline this into the rule itself?
There's an internal doc about why this is harmful, I can send you a link.